docs/Homebrew-homebrew-core-Merge-Checklist.md

```diff
@@ -1,22 +1,128 @@
-# Homebrew/homebrew-core Merge Checklist
+# Core Maintainer Guide
```

Comment:
```suggestion
# Homebrew/homebrew-core Maintainer Guide
```

Reply:
I, personally, prefer the shorter names because the longer names all look the same at first glance, but I'll go with these if that's what's preferred.